benches/serde.rs

```diff
@@ -245,3 +260,13 @@ fn bench_big_schema_read_100_record(b: &mut test::Bencher) {
 fn bench_big_schema_read_10000_record(b: &mut test::Bencher) {
     bench_read(b, &make_big_record, 10000);
 }
+
+#[bench]
+fn bench_big_schema_read_100000_record(b: &mut test::Bencher) {
+    bench_read(b, &make_big_record, 100000);
+}
+
+#[bench]
+fn bench_file_quickstop_null(b: &mut test::Bencher) {
```

Comment:
Could you add a small comment about what this file is and where it comes from?

Reply:
Yes, of course. Check the latest commit to see if the comments added are satisfactory.